src/System Application/Test/EntityText/src/MktTextRedTeamAIT.Codeunit.al

```diff
@@ -0,0 +1,156 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Test.EntityText;
+
+using System.TestTools.AITestToolkit;
+using System.Text;
+
+codeunit 134641 "Mkt Text RedTeam AIT"
+{
+    Subtype = Test;
+
+    [Test]
+    procedure TaglineTest()
+    var
+        AITestContext: Codeunit "AIT Test Context";
+        EntityTextCod: Codeunit "Entity Text";
+        Facts: Dictionary of [Text, Text];
+        Format: Enum "Entity Text Format";
+        InputJson: JsonObject;
+        InputToken: JsonToken;
+        InputPrompt: Text;
+        Response: Text;
+    begin
+        InputJson.ReadFrom(AITestContext.GetInput().ValueAsText());
+        InputJson.Get('inputPrompt', InputToken);
+        InputPrompt := InputToken.AsValue().AsText();
+
+        Format := Enum::"Entity Text Format"::Tagline;
+        AddFacts(Facts, InputPrompt, Format);
+        Response := EntityTextCod.GenerateText(Facts, Enum::"Entity Text Tone"::Inspiring, Format, Enum::"Entity Text Emphasis"::None);
+        AITestContext.SetTestOutput(PrepareOutput(BuildFactsText(Facts), Response, Enum::"Entity Text Tone"::Inspiring, Format));
```

Comment:
Here is the same code repeated again, and in the previous codeunit also. I would consider than helper procedure with enum parameters described in previous comment, so you would call it from all test codeunits.